### DISCUSSION TITLE
Support for AeroPay as a payment plugin

### DISCUSSION THREAD

**Discussion by kbrackney** [vendurehq/vendure#1733] [Q&A] [answered]
Is there any roadmap items to support additional providers for payment; for example, https://www.aeropay.com/ ?

**Comment by michaelbromley** [comment, marked as answer]
Hi,
Yes, it's quite likely that we'll expand the range of officially-supported payment providers in the future. We will start with the largest providers in the various territories that we have most users. I've not heard of AeroPay before and looking at their web site, I cannot find any public developer documentation to get an idea of how well an integration would work. 

So I'd say that if you want to support this provider, build your own handler [per the docs](https://www.vendure.io/docs/developer-guide/payment-integrations/) and if you like you can open-source it and maybe get help from community members who want to use it too.
